### PR TITLE
Rename to bbi

### PR DESCRIPTION
**Pull request by seth127**
Renaming package to bbi. Changing all mentions of babylon to bbi.

### CODE REVIEW

File: cmd/local.go [hunk 1]
```diff
@@ -5,7 +5,7 @@ import (
 	"encoding/json"
 	"errors"
 	"fmt"
-	"github.com/metrumresearchgroup/babylon/configlib"
+	"bbi/configlib"
```
[the remark this answers is not in the document]
Comment (seth127) [reply cmd/local.go:8]: @shairozan actually recommended I changed the path like this (to exclude GitHub.com). Did I understand you right with this DJ?

File: .drone.yml [hunk 2]
```diff
@@ -20,26 +20,26 @@ steps:
...
+  - name: Build bbi
     commands:
       - mkdir -p /tmp/${DRONE_BUILD_NUMBER} ; git rev-parse HEAD > /tmp/${DRONE_BUILD_NUMBER}/bbi_hash
       - go build -o bbi cmd/bbi/main.go
       - ./bbi version
-  - name: Get BabylonTest
+  - name: Get babylontest
     environment:
-      BABYLONTEST_BRANCH: master
+      babylontest_BRANCH: master
```
Comment (shairozan) [inline .drone.yml]: Even though this is purely for drone, it may make sense to just refer to this ENV as bbitest_branch instead

File: .drone.yml [hunk 3]
```diff
@@ -20,26 +20,26 @@ steps:
...
     commands:
       - git clone https://github.com/metrumresearchgroup/babylontest.git /tmp/${DRONE_BUILD_NUMBER}/babylontest
       - cd /tmp/${DRONE_BUILD_NUMBER}/babylontest
       - mkdir -p /tmp/${DRONE_BUILD_NUMBER} ; git rev-parse HEAD > /tmp/${DRONE_BUILD_NUMBER}/babylontest_hash
-      - if [ ! -z $BABYLONTEST_BRANCH ]; then git checkout $BABYLONTEST_BRANCH; fi
+      - if [ ! -z $babylontest_BRANCH ]; then git checkout $babylontest_BRANCH; fi
```
Comment (shairozan) [inline .drone.yml]: Same, change to bbitest_branch

File: cmd/local.go (hunk 1 above)
[the remark this answers is not in the document]
Comment (shairozan) [reply cmd/local.go:8]: They do not need a full path. We've done several relative path projects internally.

**Review by shairozan — APPROVED** [review]
A few more cosmetic things dealing with drone referencing Babylon, but looks good and tests are still passing.

**Pull request merged**